rules/rbe_repo.bzl

```diff
@@ -317,6 +321,10 @@ def _impl(ctx):
         project_root = project_root,
     )
 
+    # Copy all outputs to the test directory
+    if ctx.attr.create_testdata:
```

Comment:
My feeling is this calls for a refactor where the current body of the impl function goes to an inner "generate_configs" function and the _copy_to_test_dir(ctx) function is called when that returns. This way we don't risk breaking the delicate logic that requires _copy_to_test_dir(ctx) is called right before returning if additional return statements are added in a refactor.

Reply:
I agree that a refactor would be desirable, given that we only have 2 paths I'll delay doing the costly refactor until we have more paths. Also, it should be fairly easy to notice if changes don't respect the current invariant (e.g., tests would break), added a TODO.